### ISSUE TITLE
Here is only half of the functionality

### ISSUE THREAD

**Issue by 1vanK** [aaronbassett/rms-letter-sigs#10]
Now I can recognize someone I respect. But how can I find out the nobodies who opposed Stallman?

**Comment by Goury** [comment]
A fork is needed, but there's no license.
Guess why there's no license.

**Comment by alex4321** [comment]
Feel free to try this one - https://github.com/alex4321/rms-open-letter-sigs - and I hope you won't take it more seriously than to this one @1vanK @Goury  ;-)

**Comment by Goury** [comment]
@alex4321 that joke is even worse than not having a license at all.
Please take this more seriously.

**Comment by alex4321** [comment]
@Goury yeah, I also think that just parsing PR's for https://rms-open-letter.github.io/ was the best possible idea ;-)
